### PR TITLE
(RPW1-19): Create language selector dropdown

### PR DESCRIPTION
**Pull request by thibaultboursier**
# About the type of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Test plan

- [ ] Empty data
- [ ] Invalid data
- [ ] Correct data

# Checklist
### Before code review

- [x] Move Jira ticket into PR
- [x] Check with another developers that features sounds good

### After code review

- [ ] Move Jira ticket into QA

### After QA validation
If conflicts are present in github:
  - [ ] `git pull --rebase origin master`
  - [ ] Fix conflicts
  - [ ] `git rebase --continue` repeat until there is no more conflicts
  - [ ] `git push --force-with-lease`

If there's no conflicts
- [ ] squash and merge
- [ ] delete branch

### CODE REVIEW

File: src/components/Dropdown/index.js [hunk 1]
```diff
@@ -99,17 +119,36 @@ export const StyledDropdown = styled.div`
...
+    }
+    bottom: auto;
+    width: fit-content;
+  `)};
+
+  ${({ isDropdownOpen }) =>
+    isDropdownOpen &&
+    css`
+      display: block;
+    `}
```
Comment (degouville) [inline src/components/Dropdown/index.js]: Could we just indent this beautiful piece of code like this ? :

```
isDropdownOpen &&
  css`
    display: block;
  `}
```

Comment (thibaultboursier) [reply src/components/Dropdown/index.js]: It's forced by Prettier.

Comment (thibaultboursier) [reply src/components/Dropdown/index.js]: ee

File: src/components/JWPlayerETP/index.stories.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ export const getBaseProps = (id, newProps) => {
     jwplayerData: { id: jwplayerId },
     videoData: {
       provider: 'sonic',
-      id,
+      id: text('videoId', 'eurosport-e14690956c9757146ch0'),
```
Comment (MathieuDebit) [inline src/components/JWPlayerETP/index.stories.js]: Why you are not using id prop anymore?

File: src/components/Dropdown/index.js [hunk 3]
```diff
@@ -99,17 +119,36 @@ export const StyledDropdown = styled.div`
...
   }
+
+  ${small(css`
+    padding: 17px 8px 12px 17px;
+    right: -183px;
+    top: 42px;
+    @-moz-document url-prefix() {
+      right: -183px;
+    }
+    @supports (-ms-ime-align: auto) {
```
[the remark this answers is not in the document]
Comment (thibaultboursier) [reply src/components/Dropdown/index.js]: Sure, but it's legacy code I just moved.

File: src/components/JWPlayerETP/index.stories.js (hunk 2 above)
Comment (thibaultboursier) [reply src/components/JWPlayerETP/index.stories.js]: It's an error, i'm going to fix it.

File: src/components/JWPlayerETP/Controls/BottomBar/__snapshots__/index.test.js.snap [hunk 4]
```diff
@@ -196,7 +196,7 @@ exports[`Components|JWPlayerETP|Controls|BottomBar Should match snapshot 1`] = `
   display: -ms-flexbox;
   display: flex;
   height: 100%;
-  width: 72px;
+  width: 80px;
```
Comment (JulienKode) [inline src/components/JWPlayerETP/Controls/BottomBar/__snapshots__/index.test.js.snap]: Theses changes are expected on the bottom bar ?

File: src/components/JWPlayerETP/index.stories.js (hunk 2 above)
Comment (thibaultboursier) [reply src/components/JWPlayerETP/index.stories.js]: Fixed.

File: src/components/JWPlayerETP/Controls/FullScreen/index.test.js [hunk 5]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import ActionList from './index';
+import FullScreen from './index';
```
[the remark this answers is not in the document]
Comment (thibaultboursier) [reply src/components/JWPlayerETP/Controls/FullScreen/index.test.js]: I didn't understand why ActionList component had this name. It contained only fullscreen icon. I think it's more explicit to be named FullScreen, as well as we have Volume component, Slider component... These components lead to actions too.

File: src/components/Dropdown/utils/index.js [hunk 6]
```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
```
[the remark this answers is not in the document]
Comment (thibaultboursier) [reply src/components/Dropdown/utils/index.js]: Sure. In the case of this file, which could host many utils, it's sure that this rule is not useful, as there will be only named exports. But in the case where we create a component, which should be exported by default, following our conventions, it's a good thing to have this warning about default export.

File: src/components/Dropdown/index.js [hunk 7]
```diff
@@ -54,36 +57,53 @@ export const StyledArrow = styled(Chevron)`
...
+  color: ${({ theme }) => theme.dropdown.heading.color};
+  margin-bottom: 15px;
+  text-transform: uppercase;
+`;
+
+const StyledDropdownTop = () => css`
+  &::before {
+    top: auto;
+    transform: translateX(-50%) rotate(220deg);
+    bottom: -6px;
```
[the remark this answers is not in the document]
Comment (thibaultboursier) [reply src/components/Dropdown/index.js]: I agree, it's a bad practice. It's some legacy code i didn't want to refactor, but we could create another ticket to change it completely, taking example on the custom dropdown you made for Volume component.

File: src/components/Dropdown/index.js [hunk 8]
```diff
@@ -99,17 +119,36 @@ export const StyledDropdown = styled.div`
     top: -6px;
     left: 0;
   }
+
+  ${small(css`
+    padding: 17px 8px 12px 17px;
+    right: -183px;
```
[the remark this answers is not in the document]
Comment (thibaultboursier) [reply src/components/Dropdown/index.js]: Same.

File: src/components/Dropdown/utils/index.js [hunk 9]
```diff
@@ -0,0 +1,2 @@
...
+export const removeUnit = width => parseInt(width, 10);
```
[the remark this answers is not in the document]
Comment (thibaultboursier) [reply src/components/Dropdown/utils/index.js]: As we discussed, we keep this util for now, and will check for an utils library later.

File: src/components/Dropdown/utils/index.test.js [hunk 10]
```diff
@@ -0,0 +1,27 @@
+import { removeUnit } from '.';
+
+describe('utils', () => {
+  describe('removeUnit', () => {
+    it('should remove pixel unit from height', () => {
```
[the remark this answers is not in the document]
Comment (thibaultboursier) [reply src/components/Dropdown/utils/index.test.js]: Same.

**Pull request merged**